setup.py

```diff
@@ -86,6 +86,46 @@ def get_hostapd():
         sys.exit(hostapd_message)
 
 
+def get_libdbus():
+    """
+    Try to install the libdbus
+    return: None
+    retype: None
+    """
+
+    # install the libdbus-1
+    if not find_library("dbus-1"):
```

Comment:
@anakin1028 I wasn't familiar with `ctypes` and just wanted to make sure 😄 . @sophron Feel free to merge it.